content/blog/newsletter_2022-02-05.md

```diff
@@ -0,0 +1,236 @@
+
+A new stable release of [SiskinIM 7.0.1 has been published](https://github.com/tigase/siskin-im/releases/tag/7.0.1) which includes sending unencrypted messages in single chats with default encryption for OMEMO and presenting automatic file download size limit.
+
+## Servers
+
+[Openfire 4.7.0](https://discourse.igniterealtime.org/t/openfire-4-7-0-has-been-released/91268) has been released. This release is the first non-patch release in more than a year, which brings a healthy amount of new features, as well as bug fixes. Highlights of this release include extensively improved clustering support, particularly around Multi-User Chat functionality, which should benefit high-volume environments. Previously also [Openfire 4.5.5](https://discourse.igniterealtime.org/t/openfire-4-6-6-and-4-5-5-releases-log4j-only-changes/91139), [Openfire 4.6.5](https://discourse.igniterealtime.org/t/openfire-4-6-5-released/91108) and [Openfire 4.6.6](https://discourse.igniterealtime.org/t/openfire-4-6-6-and-4-5-5-releases-log4j-only-changes/91139) have been released.
+
+[Prosody 0.11.13](https://blog.prosody.im/prosody-0.11.13-released/) has been released. Since december, the new prosody releases brought some fixes to PEP to control memory usage, a security fix that addresses a denial-of-service vulnerability in Prosody’s mod_websocket, and a fix for a memory leak. Previously [Prosody 0.11.11](https://blog.prosody.im/prosody-0.11.11-released/] and [Prosody 0.11.12](https://blog.prosody.im/prosody-0.11.12-released/) were released, too.
+
+[ejabberd 21.12](https://www.process-one.net/blog/ejabberd-21-12/) has been released. The new ejabberd 21.12 release comes after five months of work, contains more than one hundred changes, many of them are major improvements or features, and several bug fixes: PubSub improvements, new mod_conversejs, and support for MUC Hats ([XEP-0317](https://xmpp.org/extensions/xep-0317.html)).
+
+[Jackal](https://github.com/ortuman/jackal), an XMPP server written in Go, had [version 0.56.0](https://github.com/ortuman/jackal/releases) released.
+
+New beta builds for the Snikket server](https://github.com/snikket-im/snikket-server/tags) have been published this month.
+
+## Libraries
+
+The [Mellium Dev Communiqué for January 2022](https://opencollective.com/mellium/updates/dev-communique-for-january-2022) has been released. Major updates this month include support for in-band bytestreams and integration testing against aioxmpp. Last month Mellium took a bit of a break, but a short update was released nevertheless. The December update can be found [here](https://opencollective.com/mellium/updates/dev-communique-for-december-2021).
```

Comment:
Oops, duplicated. Remove this!

Reply:
f41ce92397b1e9ed037405a84d0f35234edc1d83 but it does not belong into Library, correct?